handlers/handlers.go

```diff
@@ -103,6 +87,46 @@ func streamFromStorageProvider(w http.ResponseWriter, r *http.Request, clients r
 	}
 }
 
+func canGetInteractive(w http.ResponseWriter, r *http.Request, id string, clients routes.Clients, serviceAuthToken string) *interactives.Interactive {
```

Comment:
minor suggestion - `getInteractive` is better name imo

Reply:
yes.. i had the function returning a bool .. hence canXX ..